app/main.py

```diff
@@ -1,2 +1,101 @@
+from dataclasses import dataclass
+from typing import Hashable, Any, Iterator
+
+INITIAL_CAPACITY = 8
+RESIZE_THRESHOLD = 2 / 3
+CAPACITY_MULTIPLIER = 2
+
+
+@dataclass
+class Node:
```

Comment:
Hash should be also included here + would make sense to declare this class inside of Dictionary since it is part of it implementation